calico-enterprise/operations/ebpf/install.mdx

```diff
@@ -247,6 +247,16 @@ In this example, you would use `60F939227672BC3D5A1B3EC9744B2B21.gr7.us-west-2.e
 </TabItem>
 </Tabs>
 
+### Install the Tigera Operator
+
+Follow the steps in the main install for your platform that installs the Tigera Operator, without applying the custom-resources.yaml (you will update this file in a later step in this doc). If you have a KOPS cluster, you must also patch the tigera-operator deployment with DNS config so the operator can resolve the apiserver DNS.
+
+```bash
+kubectl patch deployment -n tigera-operator tigera-operator -p '{"spec":{"template":{"spec":{"dnsConfig":{"nameservers":["127.0.0.53"]}}}}}'
+```
```

Comment:
Shouldn't this be in some (sub)section  that desribes individual environments? Do we need that without ipv6?

Reply:
This is needed even for ipv4. This is applicable only for kops and no other platform. Hence I feel having sub sections is not needed. WDYT?